docs/content/guides/private-registries.md

```diff
@@ -29,9 +36,12 @@ This creates an image pull secret named `crunchy-regcred` in the `postgres-opera
 
 ## Install PGO from a Private Registry
 
-To [install PGO]({{< relref "installation/_index.md" >}}) from a private registry, you will need to set an image pull secret on the installation manifest.
+To [install PGO]({{< relref "installation/_index.md" >}}) from a private registry, you will need to
+set an image pull secret on the installation manifest.
 
-For example, to set up an image pull secret using the [Kustomize install method]({{< relref "installation/_index.md" >}}) to install PGO from the [Crunchy Data Customer Portal](https://access.crunchydata.com/), you can set the following in the `install/bases/kustomization.yaml` manifest:
+For example, to set up an image pull secret using the [Kustomize install method]({{< relref "installation/_index.md" >}})
+to install PGO from the [Crunchy Data Customer Portal](https://access.crunchydata.com/), you can set
+the following in the `install/bases/kustomization.yaml` manifest:
```

Comment:
📝 This path is changing in #3101.

Reply:
Good catch, we might as well align this with #3101.  Otherwise looks good to me as well.